### PR TITLE
Feature/massupdate extendable custom

### PR DESCRIPTION
**Pull request by PedroErnst**
Adds a helper function to load a class from custom if it is extended, otherwise from core.
Modifies calls to MassUpdate to use this helper function.

## Motivation and Context

Makes MassUpdate.php extendable through custom.

## How To Test This

1. Create a custom MassUpdate file in /custom with a class that either extends or replaces the core class.
2. See it loads correctly.

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Final checklist

- [x] My code follows the code style of this project found [here](https://docs.suitecrm.com/community/contributing-code/coding-standards/).
- [ ] My change requires a change to the documentation.
- [x] I have read the [**How to Contribute**](https://docs.suitecrm.com/community/contributing-code/) guidelines.

### CODE REVIEW

**Comment by Dillon-Brown** [comment]
Assessed

File: include/utils.php [hunk 1]
```diff
@@ -5434,18 +5434,43 @@ function isValidId($id) {
...
+
+    if (class_exists($customClassName)) {
+        return new $customClassName;
+    }
+
+    if (class_exists($className)) {
+        return new $className;
+    }
+
+    $log->fatal('Unable to find class: ' . $className);
```
Comment (gymad) [inline include/utils.php]: You can use LoggerManager::getLogger() method instead accessing to globals

**Comment by samus-aran** [comment]
Yo @PedroErnst You want to update the PR from the review? :3

File: include/utils.php (hunk 1 above)
Comment (PedroErnst) [reply include/utils.php]: Thanks for pointing this out, fixed now.

**Comment by samus-aran** [comment]
Do we need this in hotfix then? @salesagility/suitecrm-core

**Pull request merged**